.circleci/config.yml

```diff
@@ -5,6 +5,15 @@ orbs:
   browser-tools: circleci/browser-tools@1.1.1
 
 executors:
+  javascript-integration-tests:
+    docker:
+      - image: cimg/node:14.15-browsers
+      - image: thecodingmachine/gotenberg:6
+        user: gotenberg
+        environment:
+          DEFAULT_LISTEN_PORT: 3001
+    working_directory: ~/app
+
   integration-tests:
     docker:
```

Comment:
The groovy integration tests don't test PDF generation, so a running Gotenberg image isn't required.